.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
     }
   },
   "ruby.format": "rubocop",
-  "ruby.intellisense": "rubyLocate"
+  "ruby.intellisense": "rubyLocate",
+  "editor.acceptSuggestionOnEnter": "on"
```

Comment:
@KSSaiTeja You should remove this change. I think this was an un-intentional change done by your editor.